R/pkg/R/functions.R

```diff
@@ -2095,26 +2061,28 @@ setMethod("atan2", signature(y = "Column"),
             column(jc)
           })
 
-#' datediff
+#' @section Details:
+#' \code{datediff}: Returns the number of days from \code{start} to \code{end}.
 #'
-#' Returns the number of days from \code{start} to \code{end}.
-#'
-#' @param x start Column to use.
-#' @param y end Column to use.
```

Comment:
@felixcheung These names `start` and `end` are from the original doc. I now changed it to `x` and `y`.